src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -243,6 +246,16 @@ int ACA_Dataplane_OVS::update_port_state_workitem(const PortState current_PortSt
 
     try {
       assert(current_PortConfiguration.fixed_ips_size() == 1);
+      virtual_ip_address = current_PortConfiguration.fixed_ips(0).ip_address();
```

Comment:
Quick question: do we know how many times .fixed_ips(0) show up? If quite a few, suggest to "encap" it in a method which we could change it in one single place when we implement multiple fixed ips support.

Reply:
I would defer that change until we close down on the fixed_ips usage in the our DHCP and possibly router module.